### ISSUE TITLE
Text and information missing in FDK Build Notes

### ISSUE THREAD

**Issue by Arno-Enslin** [adobe-type-tools/afdko#186]
> 3.2 BooleanOperations support
> […]
> You need the source wheel in order to easily compiled the pyClipper library; else you need to

You need to do what?

--------------------------------------

I have successfully built the FDK for Windows yesterday. Well, I think, it was successfull, because it was a bit harder than one year ago. The built notes are incomplete with regard to the required compilers. On Windows you have to install Visual c express 2010 and – there I am not sure – the Microsoft Visual C++ Compiler Package for Python.

I already wrote this in other posts: The AFDKO and FontTools are damn cool. But I don't think, that it reaches many users, if the build notes are rudimentary. And the latest official FDK was released a year ago. But in the meantime many bugs were fixed. I often recommend the FDK, but for many potential users the build process seems to be too complicate.

**Comment by readroberts** [comment]
I certainly agree. The current state is because these are personal notes to me, and I never filled them out to be useful to a more general public. The good news is that the afdko is finally being reconfigured to use en external Python interpreter, and to be fully build-able and installable with the 'python setup.py install' incantation. Should be puched live within the next month or so.

**Comment by Arno-Enslin** [comment]
I have built the AFDKO again last night. And with regard to your Build notes:

1. The ufo3 versions of Defcon and Fontmath are in the Master now.

2. If you try to setup the ufo3k branch of robofab, the procedure ends with the error message, that ufo lib is missing. It was removed a year ago. I have downloaded the [ufoLib Master](https://github.com/unified-font-object/ufoLib/archive/master.zip) then and copied "ufoLib-master\Lib\ufoLib" to "robofab-ufo3k\Lib\robofab". No error message in the end of the setup procedure of robofab anymore, but I don't know, if it was the correct way to simply copy and paste that Lib.

3. You write "isntall Cython and do a lot more work."
I was able to install Cython with Pip, but I don't know, what "a lot more work" means in this context.

4. FontLab 5.2 requires a 32 bit Python. If you integrate the 64 bit version into the AFDKO, FontLab 5.2 users have to install additionally a 32 bit version.

5. Install Brotli **and Zopli.** (I have additionally [sfntly](https://github.com/daltonmaag/sfntly-py) installed. But it requires Java. I would prefer a pure Python solution for eot.)

6. I have setup [unicodedata2](https://github.com/mikekap/unicodedata2) because of the optional requirements of Fonttools.
